core/src/main/scala-2/prometheus4cats/InfoNameFromStringLiteral.scala

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I assume this will be moved to `internal` like the others?

Reply:
Yep, that's in the other PR